### PR TITLE
BGDIINF_SB-1901: Added geoadmin README and deploy script

### PR DESCRIPTION
**Pull request by ltshb**

### CODE REVIEW

File: geoadmin/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+- [GIT Branch Strategy](#git-branch-strategy)
+- [Deploy](#deploy)
+
+## GIT Branch Strategy
+
+In order to simplify the merge from the fork, we are not allowed to do any changes directly into the `master` branch,
+this branch should be kept up to date with the fork master branch.
+
+Any geoadmin changes should be done in `geoadmin_master` branch. This branch follows the general geoadmin GIT FLOW rules.
+To get the lastest changes from the radiantearth, we first merge radiantearth master branch into our master branch and
```
Comment (boecklic) [inline geoadmin/README.md:16]: We might not be able to adopt any upstream change right away. We'll probably stay on the current version until version 3 is release... maybe make a comment that new upstream versions should be thoroughly tested before merged into geoadmin_master

File: geoadmin/deploy.sh [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+echo "${YELLOW}Building with catalog ${CATALOG_URL}...${RESET}"
+STAC_VERSION=${STAC_VERSION} CATALOG_URL=${CATALOG_URL} npm run build
+
+echo "${YELLOW}Uploading to ${S3_BUCKET_NAME}...${RESET}"
+aws --profile ${AWS_PROFILE} \
+    --region eu-central-1 \
+    s3 sync \
+        --delete \
+        --cache-control "public, max-age=${MAX_AGE}" \
+        dist/  s3://${S3_BUCKET_NAME}/
```
Comment (boecklic) [inline geoadmin/deploy.sh]: does this take into account the `browser/` prefix? (see ticket)

**Review by boecklic — CHANGES_REQUESTED** [review]

File: geoadmin/deploy.sh (hunk 2 above)
Comment (ltshb) [reply geoadmin/deploy.sh]: done

File: geoadmin/README.md (hunk 1 above)
Comment (ltshb) [reply geoadmin/README.md:16]: done

**Review by boecklic — APPROVED** [review]
:1st_place_medal: except that it doesn't work yet on prod: data.geo.admin.ch/browser/index.html is showing an error

**Pull request merged**